### DISCUSSION TITLE
overlapping of Age Calculator and Whac-a-mole branch

### DISCUSSION THREAD

**Discussion by Shaba1406** [irshadmd/Awesome-CSS-Projects#171] [General]
hello, I guess there will be some issues because of the overlapping of branches the age calculator file got missing so can i make a pull request again, or you can resolve it ?

**Comment by irshadmd** [comment]
damn, how did it happen?

**Comment by irshadmd** [comment]
alright can you create a pr to add missing files

**Reply by Shaba1406** [reply]
yaa I am done with it.

**Comment by Shaba1406** [comment]
created another pull request with correct codes and files

**Reply by irshadmd** [reply]
great 🚀
